app/views/users/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<% provide(:title, 'Users') %>
+<div class="form content">
+  <h1 class="text-center">List of Users</h1>
+  <div class="container">
+    <ul class="users">
+      <% @users.each do |user| %>
+        <li>
+        <%= gravatar_for user, size:52 %>
```

Comment:
Please fix the indention like this
![image](https://user-images.githubusercontent.com/35232348/103744538-ed9fde00-5038-11eb-8ab3-f84e7b701474.png)

And then don't forget to change this to Spaces: 4